govc/USAGE.md

```diff
@@ -1838,6 +1838,7 @@ Options:
   -prefix=true           Prepend target name to image filenames if missing
   -sha=0                 Generate manifest using SHA 1, 256, 512 or 0 to skip
   -vm=                   Virtual machine [GOVC_VM]
+  -snapshot=             Specifies a snapshot to export from (supports running VMs)
```

Comment:
Did you run `make doc` to regenerate USAGE.md or add this manually?  The GH Action that checks fails due to the diff:
```diff
modified   govc/USAGE.md
@@ -1837,8 +1837,8 @@ Options:
   -name=                 Specifies target name (defaults to source name)
   -prefix=true           Prepend target name to image filenames if missing
   -sha=0                 Generate manifest using SHA 1, 256, 512 or 0 to skip
-  -vm=                   Virtual machine [GOVC_VM]
   -snapshot=             Specifies a snapshot to export from (supports running VMs)
+  -vm=                   Virtual machine [GOVC_VM]
 ```